lighthouse-core/audits/seo/is-crawlable.js

```diff
@@ -25,7 +25,7 @@ const UIStrings = {
   /** Description of a Lighthouse audit that tells the user *why* allowing search-engine crawling of their page is beneficial. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'Search engines are unable to include your pages in search results ' +
       'if they don\'t have permission to crawl them. [Learn ' +
-      'more](https://developers.google.com/web/tools/lighthouse/audits/indexing).',
+      'more](https://web.dev/is-crawable).',
```

Comment:
FYI the screenshot in this doc is of a Lighthouse error/bug instead of the audit failing